Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "fireguard"
-version = "0.1.0"
+version = "0.0.1"
```

Comment:
I am starting to push pre-release tags so the CI will build and release executables for different architectures